src/python/tensorflow_cloud/core/containerize.py

```diff
@@ -157,33 +158,6 @@ def _create_docker_file(self):
                 if float(v[0] + "." + v[1]) <= 2.1:
                     self.docker_base_image += "-py3"
 
-        if not self._base_image_exist():
-            warnings.warn(
-                "Docker base image {} does not exist.".format(
-                    self.docker_base_image))
-            if "dev" in self.docker_base_image:
-                # Except for the latest TF nightly, other nightlies
-                # do not have corresponding docker images.
-                newtag = "nightly"
-                if self.docker_base_image.endswith("-gpu"):
-                    newtag += "-gpu"
-                self.docker_base_image = (
-                    self.docker_base_image.split(":")[0] + ":" + newtag
-                )
-                warnings.warn("Using the latest TF nightly build.")
-            else:
-                warnings.warn(
-                    "Using the latest stable TF docker image available: "
-                    "`tensorflow/tensorflow:latest`"
-                    "Please see "
-                    "https://hub.docker.com/r/tensorflow/tensorflow/ "
-                    "for details on available docker images."
-                )
-                newtag = "tensorflow/tensorflow:latest"
-                if self.docker_base_image.endswith("-gpu"):
-                    newtag += "-gpu"
-                self.docker_base_image = newtag
-
         lines = [
             "FROM {}".format(self.docker_base_image),
```

Comment:
Now that you removed docker image existence check, where does error get raised if non existing image was requested? Would it yield failed Job from CMLE?

Reply:
Yes it would fail from CMLE job.